integration_tests/macro/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use proc_macro::TokenStream;
+use quote::{format_ident, quote};
+
+#[proc_macro_attribute]
+pub fn exported_test(_attr: TokenStream, item: TokenStream) -> TokenStream {
+    let input_fn = syn::parse_macro_input!(item as syn::ItemFn);
+    let fn_name = &input_fn.sig.ident;
```

Comment:
You probably want to include the module path. I.e., inside the registration function, you can write some code like:


```rust
concat!(module_path!(), "::", stringify!(#fn_name))`
```